### ISSUE TITLE
[3.1.0] pins_arduino.h not found

### ISSUE THREAD

**Issue by cyberman54** [platformio/platform-espressif32#502]
With latest v3.1.0 i get compile error with code, which did compile under v3.0.0

```
.platformio/packages/framework-arduinoespressif32/cores/esp32/Arduino.h:185:26: fatal error: 
pins_arduino.h: No such file or directory
```

**Comment by valeros** [comment]
Hi  @cyberman54 !  Please share here your configuration file.

**Comment by Qyon** [comment]
Error:
```
[env]
platform = espressif32
```

Works:
```
[env]
platform = espressif32 @ 3.0.0
```

**Comment by SQ9MDD** [comment]
Same problem here.

**Comment by valeros** [comment]
What are your board IDs?

**Comment by cyberman54** [comment]
@valeros ttgo-t-beam

**Comment by cyberman54** [comment]
Solution see https://github.com/platformio/platform-espressif32/pull/505

**Comment by valeros** [comment]
Resolved in #505
